test/suites/drift/suite_test.go

```diff
@@ -680,7 +681,7 @@ var _ = Describe("Drift", func() {
 			{
 				DeviceName: awssdk.String("/dev/xvda"),
 				EBS: &v1beta1.BlockDevice{
-					VolumeSize: resource.NewScaledQuantity(20, resource.Giga),
+					VolumeSize: resources.Quantity("20Gi"),
```

Comment:
I think we should just test each of these fields separately and this should also eliminate the need to have a separate test just for `volumeSize`

Reply:
Going to leave this the way it is since it catches a regression specifically on this volumeSize issue which cropped up a couple times only due to the "stringification" piece here. I agree that we should test each field, but I think that we can do so exclusively through the functional testing to avoid extending the time of this E2E too much